src/pages/organisation/index.tsx

```diff
@@ -203,37 +196,31 @@ const HomePage = () => (
           <InnerText title="Sustainability" theme="light">
             <>
               <p>
-                As an organisation we recognise that the escalating climate
-                crisis is the defining issue of our times.
+                Being a purposeful and sustainable business is central to the Guardian’s strategy, bringing together many aspects of our work and the decisions we take together.
               </p>
               <p>
-                In 2019, we became the first major news organisation to certify
-                as a B Corporation, and made a climate pledge to our readers
-                committing to reach net zero carbon emissions by 2030.
+                In 2019, we became the first major news organisation to certify as a B Corporation. In 2023, we successfully renewed our certification, increasing our overall score and making progress in almost every category.
               </p>
               <p>
-                In 2020 we announced we will no longer accept advertising from
-                oil and gas companies, becoming the first major global news
-                organisation to stop accepting money from companies that extract
-                fossil fuels.
+                We are committed to reaching our goal of eliminating two thirds of our emissions by 2030 and report on this annually.
               </p>
             </>
           </InnerText>
           <div css={threeColumnResponsiveCardHolder}>
             <ResponsiveCardVariant1
               title="GMG as a B Corporation"
               imagePath="/about/images/organisation-11.jpg"
-              linkUrl="https://www.theguardian.com/gnm-press-office/2019/oct/16/guardian-media-group-becomes-first-major-news-organisation-to-become-a-b-corporation-and-pledges-to-reach-net-zero-emissions-by-2030"
+              linkUrl="https://www.bcorporation.net/en-us/find-a-b-corp/company/guardian-media-group/"
             />
             <ResponsiveCardVariant1
               title="Positive impact and sustainability report"
               imagePath="/about/images/organisation-13.jpg"
-              linkUrl="https://www.theguardian.com/gmg/2021/aug/25/positive-impact-and-sustainability-report"
+              linkUrl="https://www.theguardian.com/gmg/2018/mar/14/corporate-reports-and-policies"
```

Comment:
Funny that we go from a 2021 article to a 2018 one.

Reply:
yes I believe it is intentional